index.html

```diff
@@ -0,0 +1,46 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" href="style.css">
+    <title>My porfolio</title>
+</head>
+<body>
+    <header>
+        <menu class="nav-container">
+            <h1><a href="#">Nana Antwi</a></h1>
+            <nav>
+                <img class="nav-icon" src="images/Union.png" alt="">
+            </nav>
+        </menu>
```

Comment:
- [ ] Kindly use the suggested structure provided by the previous reviewer.
```
         <header class="nav-container">
            <a href="#" class="logo">John Doe</a>
            <nav>
                <img class="nav-icon" src="images/Union.png" alt="">
            </nav>
        </header>
```
Use the appropriate tags for each element. Follow the [HTML best practices](https://github.com/microverseinc/curriculum-html-css/blob/main/articles/html_css_best_practices.md).

Reply:
Sir please  why is everyone suggesting different thing. Is like I make changes base on reviewer’s suggestions and another reviewer will come and make different suggestions

Reply:
@maccony202, it is not best practice to have two `<h1>` tags on the same HTML page. It is not good for SEO. Please refactor `<h1><a href="#">Nana Antwi</a></h1>` to `<a href="#">Nana Antwi</a>`.